app/views/pages/release-notes/29-JULY-2019.liquid

```diff
@@ -16,7 +16,8 @@ searchable: true
 
 <h4 class="release-note release-note__improved">IMPROVED</h4>
 
-* todo
+* **page slug can now consist from any components**: Previously the page slug had to be max three level deep. The improvement allows you to provide any slug you want, for example `/one/two/three/four/five`. However, please note that if you define page `/one/two/three/four` you won't be able to access `/one/two/three/four/five` - template slugs still work only for three level deep slugs.
+* **{% raw %}{{ form }}{% endraw %} output improved**: The `form` variable will not include magical keys anymore, the output now corresponds to the `fields` defined.
```

Comment:
> now corresponds to the `fields` defined.

in model schema? Or in form_configuration?

Reply:
form configuration has fields, the model schema has properties only ;)